scalalib/src/mill/scalalib/CoursierModule.scala

```diff
@@ -318,6 +368,10 @@ object CoursierModule {
      * @param resolutionParams coursier resolution parameters
      * @return dependencies with empty version filled
      */
+    @deprecated(
+      "Use resolution instead, and extract the values you're interested in from it",
+      "Mill after 0.12.9"
```

Comment:
We already know the next version.

```suggestion
      "Mill 0.12.10"
```

Reply:
Should be fixed